santosfamilyfoundation/RuinaMain
# [RUIN-195] Be able to free draw anything on the diagramming screen

# Description
The current version of the app allows the user to take a picture of a crash and save it to a report. However, a picture cannot fully capture every part of a crash accurately. The ability to draw how a crash happened and the results allow users to better understand how a crash occurred. Now a user is able to freely draw a crash diagram with their finger with multiple colors to accurately depict the crash.

# Testing
1. Start a new report
2. Fill out the quick survey
3. Click "Add Crash Diagram"
4. Draw a picture
5. Click "Save Diagram"
6. Check that diagram fully appears
7. Check that button above says "Edit Crash Diagram"
8. Click "Edit Crash Diagram"
9. Check previous diagram loads
10. Add to the diagram or start a new one
11. Save diagram
12. Export report
13. Check that both email and save to local device work (export pdf and excel)
